### PR TITLE
New package: libinstpatch-1.1.6

### PR DESCRIPTION
**Pull request by ScrelliCopter**
Optional dependency in newer versions of FluidSynth for DLS support & core dependency of the Swami project.

### CODE REVIEW

File: srcpkgs/libinstpatch/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'libinstpatch'
+pkgname=libinstpatch
+version=1.1.4
```
Comment (abenson) [inline srcpkgs/libinstpatch/template:3]: 1.1.5 is out

File: common/shlibs [hunk 2]
```diff
@@ -974,6 +974,7 @@ libfftw3f_threads.so.3 libfftw-3.3_1
 libfftw3f.so.3 libfftw-3.3_1
 libfftw3f_omp.so.3 libfftw-3.3.5_1
 libfluidsynth.so.1 libfluidsynth-1.1.5_1
+libinstpatch-1.0.so.2 libinstpatch-1.1.4_1
```
Comment (ericonr) [inline common/shlibs:977]: Is this something other packages link against? If not, it doesn't need to be here.

File: srcpkgs/libinstpatch/template (hunk 1 above)
Comment (ScrelliCopter) [reply srcpkgs/libinstpatch/template]: Updated.

File: common/shlibs (hunk 2 above)
Comment (ScrelliCopter) [reply common/shlibs]: I plan on packaging software with a dependency, should changing shlibs be done in the commit that adds the package using it?

**Comment by ScrelliCopter** [comment]
Updated to latest tagged release https://github.com/swami/libinstpatch/releases/tag/v1.1.6

**Comment by ericonr** [comment]
This was added in b82f4d545ecfe98e7ed57e8e7445c6f50f3218dd, sorry that I forgot this PR.

**Pull request closed without merging**